File: app/src/main/java/seigneur/gauvain/mycourt/ui/shotDraft/ShotDraftListCallback.java
package seigneur.gauvain.mycourt.ui.shotDraft;

import seigneur.gauvain.mycourt.data.model.Draft;

/**
 * UI callback for ShotDraft List
 */
public interface ShotDraftListCallback {

    void onShotDraftClicked(Draft shotDraft, int position);

}

File: app/src/main/java/seigneur/gauvain/mycourt/ui/shotDraft/ShotDraftListCallback.kt
package seigneur.gauvain.mycourt.ui.shotDraft

import seigneur.gauvain.mycourt.data.model.Draft

/**
 * UI callback for ShotDraft List
 */
interface ShotDraftListCallback {

    fun onShotDraftClicked(shotDraft: Draft, position: Int)

}
